frequently-asked-questions.md

```diff
@@ -20,7 +20,7 @@ CockroachDB is especially well suited for applications that require:
 
 ## How does CockroachDB scale?
 
-CockroachDB scales horizontally with minimal operator overhead. You can run it on your local computer, a single server, a corporate development cluster, or the private or public cloud. Adding capacity is as easy as pointing a new node at the running cluster. 
+CockroachDB scales horizontally with minimal operator overhead. You can run it on your local computer, a single server, a corporate development cluster, or the private or public cloud. [Adding capacity](start-a-node.html) is as easy as pointing a new node at the running cluster. 
```

Comment:
I think this should read `or a private or public cloud` as there isn't one specific private or public cloud.